tests/test_run_semantic_mask.py

```diff
@@ -30,6 +30,8 @@
     "D_proj_interp": 256,
     "train_G_ema": True,
     "dataaug_no_rotate": True,
+    "train_mask_compute_miou": True,
+    "train_mask_miou_every": 1000,
```

Comment:
This needs to be set to a very low number, e.g. 1 or 5 (to be determined) as the test dataset is too small to reach 1k iterations.